CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - manipulation SetMetricValuesWithQualityValidity
 - manipulation GenerateElementsInStates
+- manipulation SetOverflowRequest 
+- manipulation SetUnderflowRequest 
+- response SetOverOrUnderflowResponse 
```

Comment:
would suggest to only write:

manipulation SetOverflow
manipulation SetUnderflow

Reply:
reworked changelog as suggested